### PR TITLE
[#320] NotificationMenu (connects #320)

### PR DESCRIPTION
**Pull request by nirojdyola**
Added Notification sample for user login only (not for admin)

### CODE REVIEW

File: src/app/home/home.component.html [hunk 1]
```diff
@@ -21,11 +22,21 @@
...
 <!--Notification dropdown menu-->
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
-  <span mat-menu-item>Notification 1</span>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 2</div>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 3</div>
+  <span mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
+    <mat-divider></mat-divider>
+    <p [ngStyle]="{'color': 'green'}" *ngIf="notification.doc.status==='unread' && notification.doc.link">
```
Comment (lmmrssa) [inline src/app/home/home.component.html]: can we change it to some class or something that will reflect the styling

File: src/app/home/home.component.html [hunk 2]
```diff
@@ -24,10 +24,10 @@
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
   <span mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
     <mat-divider></mat-divider>
-    <p [ngStyle]="{'color': 'green'}" *ngIf="notification.doc.status==='unread' && notification.doc.link">
+    <p class="menu-Item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link">
       <a href="{{notification.doc.link}}">{{notification.doc.message}} {{notification.doc.time | date: 'MMM d, yyyy'}}</a>
     </p>
-    <p [ngStyle]="{'color': 'green'}" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">
+    <p class="menu-Item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link ===''">
```
Comment (nirojdyola) [inline src/app/home/home.component.html]: Could you please check the naming structure ? class="menu-Item-notification" should be like menu-item-notification

File: src/app/home/home.scss [hunk 3]
```diff
@@ -55,6 +55,9 @@
   }
 
 }
+.menu-item-notification {
```
Comment (razu9861) [inline src/app/home/home.scss]: indentation

File: src/app/home/home.component.html [hunk 4]
```diff
@@ -21,11 +22,21 @@
...
 <!--Notification dropdown menu-->
 <mat-menu #notificationMenu="matMenu" [overlapTrigger]="false">
-  <span mat-menu-item>Notification 1</span>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 2</div>
-  <mat-divider></mat-divider>
-  <div mat-menu-item>Notification 3</div>
+  <span mat-menu-item *ngFor="let notification of notifications" (click)="readNotification(notification.doc)">
+    <mat-divider></mat-divider>
+    <p class="menu-item-notification" *ngIf="notification.doc.status==='unread' && notification.doc.link">
```
Comment (razu9861) [inline src/app/home/home.component.html]: both read and unread message has same class. it same color will appera in both case

File: src/app/home/home.scss [hunk 5]
```diff
@@ -55,6 +55,9 @@
   }
 
 }
+.menu-Item-notification {
```
Comment (razu9861) [inline src/app/home/home.scss]: indentation

**Pull request merged**